plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcRecordSetProvider.java

```diff
@@ -100,17 +99,17 @@ public void testGetRecordSet()
         ConnectorTransactionHandle transaction = new JdbcTransactionHandle();
         JdbcRecordSetProvider recordSetProvider = new JdbcRecordSetProvider(jdbcClient, executor);
         RecordSet recordSet = recordSetProvider.getRecordSet(transaction, SESSION, split, table, ImmutableList.of(textColumn, textShortColumn, valueColumn));
-        assertNotNull(recordSet, "recordSet is null");
+        assertThat(recordSet).as("recordSet is null").isNotNull();
```

Comment:
The `as()` clause is supposed to be a description of what the test is about, not the message to print in case of failure. In particular, this assertion is about checking that recordset is not null, so the message is confusing. Same comment for all other usages of `as()`

Reply:
what should be the proper rewrite for this particular one?

Reply:
In this particular case, I would just remove the message. It doesn't add much that you can't infer from looking at the line that fails.